django INFO 2025-02-24 13:45:58,056 views 1288 34744 User asked for personal data stored: matt4
django INFO 2025-02-25 10:18:26,333 views 13908 15252 User asked for personal data stored: matt4
django INFO 2025-02-25 10:18:26,664 views 13908 15252 User logged in: testuser
django INFO 2025-02-25 10:27:41,371 views 1992 18244 User logged in: testuser
django.request WARNING 2025-02-25 10:27:41,411 log 1992 18244 Bad Request: /get_location/1/
django INFO 2025-02-25 10:27:41,736 views 1992 18244 User logged in: testuser
django.request WARNING 2025-02-25 10:27:41,761 log 1992 18244 Bad Request: /get_location/1/
django INFO 2025-02-25 10:27:42,081 views 1992 18244 User logged in: testuser
django.request WARNING 2025-02-25 10:27:42,103 log 1992 18244 Bad Request: /get_location/1/
django INFO 2025-02-25 10:27:42,432 views 1992 18244 User logged in: testuser
django INFO 2025-02-25 10:27:42,799 views 1992 18244 User logged in: testuser
django INFO 2025-02-25 10:27:43,141 views 1992 18244 User logged in: testuser
django INFO 2025-02-25 10:27:43,491 views 1992 18244 User logged in: testuser
django INFO 2025-02-25 10:27:43,841 views 1992 18244 User logged in: testuser
django INFO 2025-02-25 10:27:43,886 views 1992 18244 Logout by an unauthenticated user or session end.
django INFO 2025-02-25 10:27:45,175 views 1992 18244 User logged in: testuser
django INFO 2025-02-25 10:27:45,201 views 1992 18244 User deleted account: testuser
django INFO 2025-02-25 10:27:45,841 views 1992 18244 User logged in: testuser
django INFO 2025-02-25 10:27:45,863 views 1992 18244 User deleted account: testuser
django INFO 2025-02-25 10:27:46,516 views 1992 18244 User logged in: testuser
django INFO 2025-02-25 10:27:46,539 views 1992 18244 User deleted account: testuser
django INFO 2025-02-25 10:27:46,861 views 1992 18244 Failed login attempt for username: testuser
django INFO 2025-02-25 10:27:48,162 views 1992 18244 User logged in: testuser
django INFO 2025-02-25 10:27:50,144 views 1992 18244 User logged in: testuser
django INFO 2025-02-25 10:27:50,168 views 1992 18244 User logged out: testuser
django INFO 2025-02-25 10:27:50,841 views 1992 18244 User logged in: newuser
django INFO 2025-02-25 10:27:50,861 views 1992 18244 New user signed up: newuser
django INFO 2025-02-25 10:27:51,515 views 1992 18244 Failed login attempt for username: TESTUSER
django INFO 2025-02-25 10:27:52,165 views 1992 18244 Failed login attempt for username: TESTUSER@EXAMPLE.COM
django INFO 2025-02-25 10:27:53,461 views 1992 18244 User logged in: testuser
django WARNING 2025-02-25 10:27:53,803 views 1992 18244 Failed signup attempt.
django INFO 2025-02-25 10:27:54,776 views 1992 18244 User logged in: newuser
django INFO 2025-02-25 10:27:54,794 views 1992 18244 New user signed up: newuser
django WARNING 2025-02-25 10:27:55,121 views 1992 18244 Failed signup attempt.
django INFO 2025-02-25 10:27:55,775 views 1992 18244 Failed login attempt for username: ' OR '1'='1
django WARNING 2025-02-25 10:27:56,101 views 1992 18244 Failed signup attempt.
django WARNING 2025-02-25 10:27:56,421 views 1992 18244 Failed signup attempt.
django INFO 2025-02-25 10:27:57,063 views 1992 18244 User logged in: matt4
django INFO 2025-02-25 10:27:57,104 views 1992 18244 User asked for personal data stored: matt4
django INFO 2025-02-25 10:27:57,784 views 1992 18244 User logged in: matt4
django INFO 2025-02-25 10:27:57,834 views 1992 18244 User asked for personal data stored: matt4
django INFO 2025-02-25 10:27:58,511 views 1992 18244 User logged in: matt4
django INFO 2025-02-25 10:27:58,551 views 1992 18244 User asked for personal data stored: matt4
django INFO 2025-02-25 10:27:58,913 views 1992 18244 User logged in: testuser
django INFO 2025-02-25 10:28:48,465 views 31236 27216 User logged in: testuser
django.request WARNING 2025-02-25 10:28:48,500 log 31236 27216 Bad Request: /get_location/1/
django INFO 2025-02-25 10:28:48,827 views 31236 27216 User logged in: testuser
django.request WARNING 2025-02-25 10:28:48,856 log 31236 27216 Bad Request: /get_location/1/
django INFO 2025-02-25 10:28:49,188 views 31236 27216 User logged in: testuser
django.request WARNING 2025-02-25 10:28:49,211 log 31236 27216 Bad Request: /get_location/1/
django INFO 2025-02-25 10:28:49,527 views 31236 27216 User logged in: testuser
django INFO 2025-02-25 10:28:49,896 views 31236 27216 User logged in: testuser
django INFO 2025-02-25 10:28:50,256 views 31236 27216 User logged in: testuser
django INFO 2025-02-25 10:28:50,616 views 31236 27216 User logged in: testuser
django INFO 2025-02-25 10:28:50,668 views 31236 27216 Logout by an unauthenticated user or session end.
django INFO 2025-02-25 10:28:52,001 views 31236 27216 User logged in: testuser
django INFO 2025-02-25 10:28:52,029 views 31236 27216 User deleted account: testuser
django INFO 2025-02-25 10:28:52,671 views 31236 27216 User logged in: testuser
django INFO 2025-02-25 10:28:52,691 views 31236 27216 User deleted account: testuser
django INFO 2025-02-25 10:28:53,331 views 31236 27216 User logged in: testuser
django INFO 2025-02-25 10:28:53,361 views 31236 27216 User deleted account: testuser
django INFO 2025-02-25 10:28:53,686 views 31236 27216 Failed login attempt for username: testuser
django INFO 2025-02-25 10:28:54,981 views 31236 27216 User logged in: testuser
django INFO 2025-02-25 10:28:56,961 views 31236 27216 User logged in: testuser
django INFO 2025-02-25 10:28:56,985 views 31236 27216 User logged out: testuser
django INFO 2025-02-25 10:28:57,632 views 31236 27216 User logged in: newuser
django INFO 2025-02-25 10:28:57,651 views 31236 27216 New user signed up: newuser
django INFO 2025-02-25 10:28:58,294 views 31236 27216 Failed login attempt for username: TESTUSER
django INFO 2025-02-25 10:28:58,945 views 31236 27216 Failed login attempt for username: TESTUSER@EXAMPLE.COM
django INFO 2025-02-25 10:29:00,222 views 31236 27216 User logged in: testuser
django WARNING 2025-02-25 10:29:00,573 views 31236 27216 Failed signup attempt.
django INFO 2025-02-25 10:29:01,531 views 31236 27216 User logged in: newuser
django INFO 2025-02-25 10:29:01,553 views 31236 27216 New user signed up: newuser
django WARNING 2025-02-25 10:29:01,881 views 31236 27216 Failed signup attempt.
django INFO 2025-02-25 10:29:02,521 views 31236 27216 Failed login attempt for username: ' OR '1'='1
django WARNING 2025-02-25 10:29:02,846 views 31236 27216 Failed signup attempt.
django WARNING 2025-02-25 10:29:03,163 views 31236 27216 Failed signup attempt.
django INFO 2025-02-25 10:29:03,802 views 31236 27216 User logged in: matt4
django INFO 2025-02-25 10:29:03,844 views 31236 27216 User asked for personal data stored: matt4
django INFO 2025-02-25 10:29:04,524 views 31236 27216 User logged in: matt4
django INFO 2025-02-25 10:29:04,574 views 31236 27216 User asked for personal data stored: matt4
django INFO 2025-02-25 10:29:05,253 views 31236 27216 User logged in: matt4
django INFO 2025-02-25 10:29:05,291 views 31236 27216 User asked for personal data stored: matt4
django INFO 2025-02-25 10:29:05,650 views 31236 27216 User logged in: testuser
django.utils.autoreload INFO 2025-02-25 10:31:25,894 autoreload 6152 11932 Watching for file changes with StatReloader
django.server INFO 2025-02-25 10:31:27,423 basehttp 6152 3888 "GET / HTTP/1.1" 302 0
django.server INFO 2025-02-25 10:31:27,485 basehttp 6152 3888 "GET /accounts/login/ HTTP/1.1" 200 1424
django.server WARNING 2025-02-25 10:31:27,603 basehttp 6152 3888 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-02-25 10:31:32,804 views 6152 33632 User logged in: matt
django.server INFO 2025-02-25 10:31:32,922 basehttp 6152 33632 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-02-25 10:31:32,967 basehttp 6152 33632 "GET /home/ HTTP/1.1" 200 1682
django.server WARNING 2025-02-25 10:31:33,056 basehttp 6152 33632 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-02-25 10:31:38,302 basehttp 6152 12928 "GET /leaderboard/ HTTP/1.1" 200 608
django.server INFO 2025-02-25 10:31:38,473 basehttp 6152 12928 "GET /static/leaderboard/styles.css HTTP/1.1" 200 725
django.server INFO 2025-02-25 10:31:45,331 basehttp 6152 12928 "GET /tasks/ HTTP/1.1" 200 3439
django.server WARNING 2025-02-25 10:31:45,385 basehttp 6152 12928 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-02-25 10:31:48,682 basehttp 6152 32576 "GET /get_location/1/? HTTP/1.1" 200 2198
django.server INFO 2025-02-25 10:31:48,735 basehttp 6152 32576 "GET /static/css/button.css HTTP/1.1" 304 0
django.server INFO 2025-02-25 10:31:50,747 basehttp 6152 32576 "GET /get_location/1/?lat=50.7309945&lon=-3.5420185 HTTP/1.1" 200 2069
django.server INFO 2025-02-25 10:31:55,816 basehttp 6152 32576 "GET /tasks/ HTTP/1.1" 200 3439
django.server WARNING 2025-02-25 10:31:55,863 basehttp 6152 32576 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-02-25 10:32:01,205 basehttp 6152 8884 "GET /home/ HTTP/1.1" 200 1682
django.server WARNING 2025-02-25 10:32:01,261 basehttp 6152 8884 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-02-25 10:32:05,423 views 6152 9008 User asked for personal data stored: matt
django.server INFO 2025-02-25 10:32:05,442 basehttp 6152 9008 "GET /home/accounts/view_data/ HTTP/1.1" 200 1316
django.server WARNING 2025-02-25 10:32:05,491 basehttp 6152 9008 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-02-25 10:32:13,208 basehttp 6152 10572 "GET /home/ HTTP/1.1" 200 1682
django.server WARNING 2025-02-25 10:32:13,277 basehttp 6152 10572 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-02-25 10:32:16,880 basehttp 6152 13996 "GET /home/accounts/update_profile/ HTTP/1.1" 200 1521
django.server WARNING 2025-02-25 10:32:16,931 basehttp 6152 13996 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-02-25 10:32:19,359 views 6152 30884 User updated profile (username & email): matt123
django.server INFO 2025-02-25 10:32:19,360 basehttp 6152 30884 "POST /home/accounts/update_profile/ HTTP/1.1" 302 0
django.server INFO 2025-02-25 10:32:19,384 basehttp 6152 30884 "GET /home/ HTTP/1.1" 200 1685
django.server WARNING 2025-02-25 10:32:19,439 basehttp 6152 30884 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-02-25 10:32:20,914 views 6152 21280 User asked for personal data stored: matt123
django.server INFO 2025-02-25 10:32:20,932 basehttp 6152 21280 "GET /home/accounts/view_data/ HTTP/1.1" 200 1355
django.server WARNING 2025-02-25 10:32:20,984 basehttp 6152 21280 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-02-25 10:32:26,855 basehttp 6152 7580 "GET /home/ HTTP/1.1" 200 1685
django.server WARNING 2025-02-25 10:32:26,933 basehttp 6152 7580 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-02-25 10:32:27,581 views 6152 33636 User logged out: matt123
django.server INFO 2025-02-25 10:32:27,581 basehttp 6152 33636 "GET /home/accounts/logout/ HTTP/1.1" 302 0
django.server INFO 2025-02-25 10:32:27,588 basehttp 6152 33636 "GET /home/accounts/login/ HTTP/1.1" 200 1424
django.server WARNING 2025-02-25 10:32:27,666 basehttp 6152 33636 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-02-25 10:34:04,970 autoreload 32424 26292 Watching for file changes with StatReloader
django.server INFO 2025-02-25 10:34:08,203 basehttp 32424 20600 "GET / HTTP/1.1" 302 0
django.server INFO 2025-02-25 10:34:08,246 basehttp 32424 20600 "GET /accounts/login/ HTTP/1.1" 200 1424
django.server WARNING 2025-02-25 10:34:08,349 basehttp 32424 20600 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-02-25 10:34:16,384 views 32424 26132 User logged in: dc754
django.server INFO 2025-02-25 10:34:16,466 basehttp 32424 26132 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-02-25 10:34:16,506 basehttp 32424 26132 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-02-25 10:34:16,554 basehttp 32424 26132 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-02-25 10:34:18,192 basehttp 32424 26520 "GET /PrivacyPolicy/ HTTP/1.1" 200 72592
django.utils.autoreload INFO 2025-02-25 10:36:25,646 autoreload 32828 8920 Watching for file changes with StatReloader
django.server INFO 2025-02-25 10:36:27,353 basehttp 32828 21672 "GET / HTTP/1.1" 302 0
django.server INFO 2025-02-25 10:36:27,402 basehttp 32828 21672 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-02-25 10:36:27,500 basehttp 32828 21672 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-02-25 10:36:30,799 basehttp 32828 17860 "GET /TermsAndConditions/ HTTP/1.1" 200 34408
django.server INFO 2025-02-25 10:36:32,793 basehttp 32828 17860 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-02-25 10:36:34,106 basehttp 32828 17860 "GET /PrivacyPolicy/ HTTP/1.1" 200 72592
django.server INFO 2025-02-25 10:36:35,285 basehttp 32828 17860 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-02-25 10:36:38,299 basehttp 32828 17860 "GET /home/signup/ HTTP/1.1" 200 2526
django.server WARNING 2025-02-25 10:36:38,349 basehttp 32828 17860 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-02-25 10:36:39,652 basehttp 32828 10572 "GET /PrivacyPolicy/ HTTP/1.1" 200 72592
django.server INFO 2025-02-25 10:36:42,199 basehttp 32828 10572 "GET /TermsAndConditions/ HTTP/1.1" 200 34408
django.utils.autoreload INFO 2025-03-03 14:03:15,193 autoreload 33424 3128 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-03 14:04:14,858 autoreload 2024 29368 Watching for file changes with StatReloader
django.server INFO 2025-03-03 14:04:16,208 basehttp 2024 9932 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-03 14:04:16,333 basehttp 2024 9932 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-03 14:04:16,393 basehttp 2024 9932 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-03 14:04:18,384 views 2024 27908 User logged in: dc754
django.server INFO 2025-03-03 14:04:18,406 basehttp 2024 27908 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-03 14:04:18,416 basehttp 2024 27908 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-03 14:04:18,448 basehttp 2024 27908 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-03 14:04:19,993 basehttp 2024 5840 "GET /tasks/ HTTP/1.1" 200 10750
django.server WARNING 2025-03-03 14:04:20,026 basehttp 2024 5840 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-03 14:04:37,484 basehttp 2024 4396 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-03 14:04:37,530 basehttp 2024 4396 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-03 14:18:34,172 autoreload 5332 24476 Watching for file changes with StatReloader
django.server INFO 2025-03-03 14:18:35,963 basehttp 5332 20140 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-03 14:18:35,991 basehttp 5332 20140 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-03 14:18:36,028 basehttp 5332 20140 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-03 14:18:37,486 views 5332 29340 User logged in: dc754
django.server INFO 2025-03-03 14:18:37,508 basehttp 5332 29340 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-03 14:18:37,521 basehttp 5332 29340 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-03 14:18:37,557 basehttp 5332 29340 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-03 14:18:38,924 basehttp 5332 14788 "GET /tasks/ HTTP/1.1" 200 10750
django.server WARNING 2025-03-03 14:18:38,957 basehttp 5332 14788 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-03 14:18:42,910 basehttp 5332 3096 "GET /get_location/3/? HTTP/1.1" 200 2198
django.server INFO 2025-03-03 14:18:42,940 basehttp 5332 3096 "GET /static/css/button.css HTTP/1.1" 304 0
django.server INFO 2025-03-03 14:18:48,095 basehttp 5332 3096 "GET /get_location/3/?lat=50.730677762429345&lon=-3.5421281910313964 HTTP/1.1" 200 2069
django.server INFO 2025-03-03 14:18:55,138 basehttp 5332 3096 "GET /tasks/ HTTP/1.1" 200 10750
django.server WARNING 2025-03-03 14:18:55,167 basehttp 5332 3096 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-03 14:42:58,262 autoreload 10508 27212 Watching for file changes with StatReloader
django.server INFO 2025-03-03 14:42:59,370 basehttp 10508 30592 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-03 14:42:59,414 basehttp 10508 30592 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-03 14:42:59,461 basehttp 10508 30592 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-03 14:43:00,863 views 10508 30084 User logged in: dc754
django.server INFO 2025-03-03 14:43:00,890 basehttp 10508 30084 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-03 14:43:00,902 basehttp 10508 30084 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-03 14:43:00,933 basehttp 10508 30084 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.request ERROR 2025-03-03 14:43:01,933 log 10508 32732 Internal Server Error: /tasks/
Traceback (most recent call last):
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\dc754\OneDrive\Documents\year2\softwareEngineering\Group-Engineering-Project\SustainableApplication\tasks\views.py", line 22, in tasks_view
    if selected_task.last_updated.date() != today:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Tasks' object has no attribute 'last_updated'
django.server ERROR 2025-03-03 14:43:01,936 basehttp 10508 32732 "GET /tasks/ HTTP/1.1" 500 70348
django.utils.autoreload INFO 2025-03-03 14:43:44,152 autoreload 10508 27212 C:\Users\dc754\OneDrive\Documents\year2\softwareEngineering\Group-Engineering-Project\SustainableApplication\tasks\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-03 14:43:44,804 autoreload 23316 27632 Watching for file changes with StatReloader
django.server INFO 2025-03-03 14:43:46,001 basehttp 23316 3620 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-03 14:43:46,033 basehttp 23316 3620 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-03 14:43:46,079 basehttp 23316 3620 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-03 14:43:47,417 views 23316 27676 User logged in: dc754
django.server INFO 2025-03-03 14:43:47,442 basehttp 23316 27676 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-03 14:43:47,453 basehttp 23316 27676 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-03 14:43:47,489 basehttp 23316 27676 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-03 14:43:48,233 basehttp 23316 23852 "GET /tasks/ HTTP/1.1" 200 2725
django.server WARNING 2025-03-03 14:43:48,270 basehttp 23316 23852 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-04 00:06:44,621 autoreload 7128 8596 Watching for file changes with StatReloader
django.server INFO 2025-03-04 00:06:46,545 basehttp 7128 29276 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-04 00:06:46,841 basehttp 7128 29276 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-04 00:06:46,948 basehttp 7128 29276 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-04 00:06:48,846 views 7128 15536 User logged in: dc754
django.server INFO 2025-03-04 00:06:48,895 basehttp 7128 15536 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-04 00:06:48,939 basehttp 7128 15536 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-04 00:06:49,015 basehttp 7128 15536 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-04 00:06:50,171 basehttp 7128 18480 "GET /tasks/ HTTP/1.1" 200 2725
django.server WARNING 2025-03-04 00:06:50,247 basehttp 7128 18480 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-04 11:13:08,182 autoreload 31656 11480 Watching for file changes with StatReloader
django.server INFO 2025-03-04 11:13:09,724 basehttp 31656 31464 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-04 11:13:09,795 basehttp 31656 31464 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-04 11:13:09,913 basehttp 31656 31464 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-04 11:13:11,874 views 31656 28532 User logged in: dc754
django.server INFO 2025-03-04 11:13:11,918 basehttp 31656 28532 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-04 11:13:11,936 basehttp 31656 28532 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-04 11:13:11,989 basehttp 31656 28532 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-04 11:13:13,179 basehttp 31656 11220 "GET /tasks/ HTTP/1.1" 200 2725
django.server WARNING 2025-03-04 11:13:13,266 basehttp 31656 11220 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-04 11:13:17,770 basehttp 31656 30448 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-04 11:13:18,004 basehttp 31656 30448 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-08 20:33:39,285 autoreload 21172 38820 Watching for file changes with StatReloader
django.server INFO 2025-03-08 20:33:40,930 basehttp 21172 10312 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:33:41,079 basehttp 21172 10312 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-08 20:33:41,179 basehttp 21172 10312 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.request WARNING 2025-03-08 20:33:41,338 log 21172 18392 Not Found: /favicon.ico
django.server WARNING 2025-03-08 20:33:41,340 basehttp 21172 18392 "GET /favicon.ico HTTP/1.1" 404 4253
django INFO 2025-03-08 20:33:43,071 views 21172 18392 User logged in: dc754
django.server INFO 2025-03-08 20:33:43,099 basehttp 21172 18392 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:33:43,118 basehttp 21172 18392 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:33:43,149 basehttp 21172 18392 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-08 20:33:45,644 basehttp 21172 27128 "GET /tasks/ HTTP/1.1" 200 2725
django.server WARNING 2025-03-08 20:33:45,667 basehttp 21172 27128 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-08 20:34:19,280 autoreload 18856 27076 Watching for file changes with StatReloader
django.server INFO 2025-03-08 20:34:20,930 basehttp 18856 23612 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:34:20,967 basehttp 18856 23612 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-08 20:34:21,004 basehttp 18856 23612 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-08 20:34:22,101 views 18856 37024 User logged in: dc754
django.server INFO 2025-03-08 20:34:22,124 basehttp 18856 37024 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:34:22,133 basehttp 18856 37024 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:34:22,165 basehttp 18856 37024 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-08 20:34:23,004 basehttp 18856 23164 "GET /tasks/ HTTP/1.1" 200 2725
django.server WARNING 2025-03-08 20:34:23,038 basehttp 18856 23164 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-08 20:35:18,726 autoreload 34804 2980 Watching for file changes with StatReloader
django.server INFO 2025-03-08 20:35:19,757 basehttp 34804 17788 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:35:19,788 basehttp 34804 17788 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-08 20:35:19,827 basehttp 34804 17788 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-08 20:35:21,110 views 34804 28892 User logged in: dc754
django.server INFO 2025-03-08 20:35:21,133 basehttp 34804 28892 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:35:21,143 basehttp 34804 28892 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:35:21,169 basehttp 34804 28892 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-08 20:35:21,836 basehttp 34804 19604 "GET /tasks/ HTTP/1.1" 200 2725
django.server WARNING 2025-03-08 20:35:21,868 basehttp 34804 19604 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-08 20:38:30,147 autoreload 14068 14564 Watching for file changes with StatReloader
django.server INFO 2025-03-08 20:38:32,243 basehttp 14068 21736 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:38:32,279 basehttp 14068 21736 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-08 20:38:32,319 basehttp 14068 21736 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-08 20:38:34,130 views 14068 36004 User logged in: dc754
django.server INFO 2025-03-08 20:38:34,154 basehttp 14068 36004 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:38:34,164 basehttp 14068 36004 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:38:34,190 basehttp 14068 36004 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.request ERROR 2025-03-08 20:38:35,482 log 14068 22692 Internal Server Error: /tasks/
Traceback (most recent call last):
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\dc754\OneDrive\Documents\year2\softwareEngineering\Group-Engineering-Project\SustainableApplication\tasks\views.py", line 14, in tasks_view
    print(incomplete_tasks.count())
          ~~~~~~~~~~~~~~~~~~~~~~^^
TypeError: list.count() takes exactly one argument (0 given)
django.server ERROR 2025-03-08 20:38:35,484 basehttp 14068 22692 "GET /tasks/ HTTP/1.1" 500 69946
django.utils.autoreload INFO 2025-03-08 20:38:47,067 autoreload 14068 14564 C:\Users\dc754\OneDrive\Documents\year2\softwareEngineering\Group-Engineering-Project\SustainableApplication\tasks\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-08 20:38:47,787 autoreload 36844 2436 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-08 20:39:03,394 autoreload 15372 31720 Watching for file changes with StatReloader
django.server INFO 2025-03-08 20:39:07,158 basehttp 15372 3672 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:39:07,187 basehttp 15372 3672 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-08 20:39:07,226 basehttp 15372 3672 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-08 20:39:08,611 views 15372 14028 User logged in: dc754
django.server INFO 2025-03-08 20:39:08,634 basehttp 15372 14028 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:39:08,644 basehttp 15372 14028 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:39:08,670 basehttp 15372 14028 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-08 20:39:09,463 basehttp 15372 37364 "GET /tasks/ HTTP/1.1" 200 2725
django.server WARNING 2025-03-08 20:39:09,490 basehttp 15372 37364 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-08 20:41:29,201 autoreload 25800 37000 Watching for file changes with StatReloader
django.server INFO 2025-03-08 20:41:30,269 basehttp 25800 38792 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:41:30,296 basehttp 25800 38792 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-08 20:41:30,336 basehttp 25800 38792 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-08 20:41:31,717 views 25800 33960 User logged in: dc754
django.server INFO 2025-03-08 20:41:31,740 basehttp 25800 33960 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:41:31,751 basehttp 25800 33960 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:41:31,776 basehttp 25800 33960 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-08 20:41:32,860 basehttp 25800 24204 "GET /tasks/ HTTP/1.1" 200 2725
django.server WARNING 2025-03-08 20:41:32,887 basehttp 25800 24204 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-08 20:42:31,708 autoreload 25800 37000 C:\Users\dc754\OneDrive\Documents\year2\softwareEngineering\Group-Engineering-Project\SustainableApplication\tasks\models.py changed, reloading.
django.utils.autoreload INFO 2025-03-08 20:42:32,350 autoreload 36000 25080 Watching for file changes with StatReloader
django.server INFO 2025-03-08 20:42:34,177 basehttp 36000 16380 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:42:34,213 basehttp 36000 16380 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-08 20:42:34,261 basehttp 36000 16380 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-08 20:42:35,801 views 36000 23344 User logged in: dc754
django.server INFO 2025-03-08 20:42:35,825 basehttp 36000 23344 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:42:35,836 basehttp 36000 23344 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:42:35,872 basehttp 36000 23344 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-08 20:42:36,889 basehttp 36000 24004 "GET /tasks/ HTTP/1.1" 200 2725
django.server WARNING 2025-03-08 20:42:36,919 basehttp 36000 24004 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-08 20:43:54,050 autoreload 28060 12632 Watching for file changes with StatReloader
django.server INFO 2025-03-08 20:43:55,193 basehttp 28060 19392 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:43:55,231 basehttp 28060 19392 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-08 20:43:55,271 basehttp 28060 19392 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-08 20:43:56,652 views 28060 22776 User logged in: dc754
django.server INFO 2025-03-08 20:43:56,678 basehttp 28060 22776 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:43:56,689 basehttp 28060 22776 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:43:56,726 basehttp 28060 22776 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-08 20:43:57,507 basehttp 28060 19500 "GET /tasks/ HTTP/1.1" 200 2725
django.server WARNING 2025-03-08 20:43:57,535 basehttp 28060 19500 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-08 20:47:50,299 autoreload 33160 23992 Watching for file changes with StatReloader
django.server INFO 2025-03-08 20:47:51,303 basehttp 33160 9420 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:47:51,332 basehttp 33160 9420 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-08 20:47:51,389 basehttp 33160 9420 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-08 20:47:52,731 views 33160 34320 User logged in: dc754
django.server INFO 2025-03-08 20:47:52,756 basehttp 33160 34320 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:47:52,768 basehttp 33160 34320 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:47:52,808 basehttp 33160 34320 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-08 20:47:53,743 basehttp 33160 20176 "GET /tasks/ HTTP/1.1" 200 2725
django.server WARNING 2025-03-08 20:47:53,771 basehttp 33160 20176 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-08 20:50:22,540 autoreload 11520 14148 Watching for file changes with StatReloader
django.server INFO 2025-03-08 20:50:23,915 basehttp 11520 12320 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:50:23,949 basehttp 11520 12320 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-08 20:50:23,987 basehttp 11520 12320 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-08 20:50:25,497 views 11520 38812 User logged in: dc754
django.server INFO 2025-03-08 20:50:25,522 basehttp 11520 38812 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:50:25,533 basehttp 11520 38812 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:50:25,559 basehttp 11520 38812 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-08 20:50:26,663 basehttp 11520 13416 "GET /tasks/ HTTP/1.1" 200 3713
django.server WARNING 2025-03-08 20:50:26,691 basehttp 11520 13416 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-08 20:50:35,671 basehttp 11520 37696 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:50:35,696 basehttp 11520 37696 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-08 20:51:10,765 autoreload 35124 36892 Watching for file changes with StatReloader
django.server INFO 2025-03-08 20:51:12,280 basehttp 35124 8468 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:51:12,306 basehttp 35124 8468 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-08 20:51:12,345 basehttp 35124 8468 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-08 20:51:13,882 views 35124 36476 User logged in: dc754
django.server INFO 2025-03-08 20:51:13,907 basehttp 35124 36476 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-08 20:51:13,917 basehttp 35124 36476 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-08 20:51:13,945 basehttp 35124 36476 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-08 20:51:14,882 basehttp 35124 22728 "GET /tasks/ HTTP/1.1" 200 3709
django.server WARNING 2025-03-08 20:51:14,909 basehttp 35124 22728 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-13 10:13:55,833 autoreload 19512 24104 Watching for file changes with StatReloader
django.server INFO 2025-03-13 10:13:58,269 basehttp 19512 19552 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:13:58,501 basehttp 19512 19552 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-13 10:13:58,618 basehttp 19512 19552 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.request WARNING 2025-03-13 10:13:59,209 log 19512 23204 Not Found: /favicon.ico
django.server WARNING 2025-03-13 10:13:59,210 basehttp 19512 23204 "GET /favicon.ico HTTP/1.1" 404 4253
django INFO 2025-03-13 10:14:01,320 views 19512 23204 User logged in: dc754
django.server INFO 2025-03-13 10:14:01,372 basehttp 19512 23204 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:14:01,409 basehttp 19512 23204 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-13 10:14:01,471 basehttp 19512 23204 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.request ERROR 2025-03-13 10:14:04,469 log 19512 26804 Internal Server Error: /tasks/
Traceback (most recent call last):
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: tasks_tasks.user_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\dc754\OneDrive\Documents\year2\softwareEngineering\Group-Engineering-Project\SustainableApplication\tasks\views.py", line 18, in tasks_view
    user_incomplete_tasks = list(Tasks.objects.filter(completed=False))
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 400, in __iter__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: tasks_tasks.user_id
django.server ERROR 2025-03-13 10:14:04,487 basehttp 19512 26804 "GET /tasks/ HTTP/1.1" 500 134838
django.utils.autoreload INFO 2025-03-13 10:20:55,179 autoreload 25344 23048 Watching for file changes with StatReloader
django.server INFO 2025-03-13 10:20:56,596 basehttp 25344 22476 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:20:56,656 basehttp 25344 22476 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-13 10:20:56,737 basehttp 25344 22476 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-13 10:20:58,555 views 25344 24568 User logged in: dc754
django.server INFO 2025-03-13 10:20:58,621 basehttp 25344 24568 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:20:58,644 basehttp 25344 24568 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-13 10:20:58,721 basehttp 25344 24568 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-13 10:20:59,778 basehttp 25344 26400 "GET /tasks/ HTTP/1.1" 200 3713
django.server WARNING 2025-03-13 10:20:59,824 basehttp 25344 26400 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-13 10:21:12,521 autoreload 6536 3032 Watching for file changes with StatReloader
django.server INFO 2025-03-13 10:21:13,986 basehttp 6536 15120 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:21:14,046 basehttp 6536 15120 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-13 10:21:14,123 basehttp 6536 15120 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-13 10:21:16,045 views 6536 3276 User logged in: dc754
django.server INFO 2025-03-13 10:21:16,105 basehttp 6536 3276 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:21:16,128 basehttp 6536 3276 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-13 10:21:16,198 basehttp 6536 3276 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-13 10:21:17,616 basehttp 6536 23348 "GET /tasks/ HTTP/1.1" 200 3713
django.server WARNING 2025-03-13 10:21:17,671 basehttp 6536 23348 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-13 10:24:16,045 autoreload 11040 16000 Watching for file changes with StatReloader
django.server INFO 2025-03-13 10:24:17,492 basehttp 11040 26980 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:24:17,703 basehttp 11040 26980 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-13 10:24:17,802 basehttp 11040 26980 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-13 10:24:19,735 views 11040 20916 User logged in: dc754
django.server INFO 2025-03-13 10:24:19,783 basehttp 11040 20916 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:24:19,819 basehttp 11040 20916 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-13 10:24:19,894 basehttp 11040 20916 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-13 10:24:21,153 basehttp 11040 26924 "GET /tasks/ HTTP/1.1" 200 3707
django.server WARNING 2025-03-13 10:24:21,202 basehttp 11040 26924 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-13 10:26:48,013 autoreload 14424 27512 Watching for file changes with StatReloader
django.server INFO 2025-03-13 10:26:49,474 basehttp 14424 5304 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:26:49,535 basehttp 14424 5304 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-13 10:26:49,632 basehttp 14424 5304 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-13 10:26:51,510 views 14424 22464 User logged in: dc754
django.server INFO 2025-03-13 10:26:51,570 basehttp 14424 22464 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:26:51,594 basehttp 14424 22464 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-13 10:26:51,668 basehttp 14424 22464 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-13 10:26:53,098 basehttp 14424 20128 "GET /tasks/ HTTP/1.1" 200 3721
django.server WARNING 2025-03-13 10:26:53,149 basehttp 14424 20128 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-13 10:29:39,903 autoreload 14424 27512 C:\Users\dc754\OneDrive\Documents\year2\softwareEngineering\Group-Engineering-Project\SustainableApplication\home\models.py changed, reloading.
django.utils.autoreload INFO 2025-03-13 10:29:41,908 autoreload 25416 22836 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-13 10:29:43,702 autoreload 25416 22836 C:\Users\dc754\OneDrive\Documents\year2\softwareEngineering\Group-Engineering-Project\SustainableApplication\home\models.py changed, reloading.
django.utils.autoreload INFO 2025-03-13 10:29:45,290 autoreload 11744 27204 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-13 10:32:37,134 autoreload 11268 22236 Watching for file changes with StatReloader
django.server INFO 2025-03-13 10:32:38,611 basehttp 11268 26860 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:32:38,666 basehttp 11268 26860 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-13 10:32:38,753 basehttp 11268 26860 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-13 10:32:40,889 views 11268 26800 User logged in: dc754
django.server INFO 2025-03-13 10:32:40,936 basehttp 11268 26800 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:32:40,956 basehttp 11268 26800 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-13 10:32:41,016 basehttp 11268 26800 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-13 10:32:42,300 basehttp 11268 12264 "GET /tasks/ HTTP/1.1" 200 3713
django.server WARNING 2025-03-13 10:32:42,355 basehttp 11268 12264 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-13 10:32:52,959 autoreload 1976 16600 Watching for file changes with StatReloader
django.server INFO 2025-03-13 10:32:54,389 basehttp 1976 15892 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:32:54,450 basehttp 1976 15892 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-13 10:32:54,542 basehttp 1976 15892 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-13 10:32:56,989 views 1976 25656 User logged in: dc754
django.server INFO 2025-03-13 10:32:57,034 basehttp 1976 25656 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:32:57,055 basehttp 1976 25656 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-13 10:32:57,116 basehttp 1976 25656 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.request ERROR 2025-03-13 10:32:58,274 log 1976 11876 Internal Server Error: /tasks/
Traceback (most recent call last):
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\dc754\OneDrive\Documents\year2\softwareEngineering\Group-Engineering-Project\SustainableApplication\tasks\views.py", line 21, in tasks_view
    if user.selected_task and user.task_assing_date == today:
                              ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\dc754\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
AttributeError: 'CustomUser' object has no attribute 'task_assing_date'. Did you mean: 'task_assign_date'?
django.server ERROR 2025-03-13 10:32:58,297 basehttp 1976 11876 "GET /tasks/ HTTP/1.1" 500 74412
django.utils.autoreload INFO 2025-03-13 10:33:13,393 autoreload 1976 16600 C:\Users\dc754\OneDrive\Documents\year2\softwareEngineering\Group-Engineering-Project\SustainableApplication\tasks\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-13 10:33:14,930 autoreload 16944 6300 Watching for file changes with StatReloader
django.server INFO 2025-03-13 10:33:16,662 basehttp 16944 13984 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:33:16,739 basehttp 16944 13984 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server WARNING 2025-03-13 10:33:16,875 basehttp 16944 13984 "GET /static/css/styles.css HTTP/1.1" 404 1969
django INFO 2025-03-13 10:33:18,605 views 16944 22464 User logged in: dc754
django.server INFO 2025-03-13 10:33:18,667 basehttp 16944 22464 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-13 10:33:18,687 basehttp 16944 22464 "GET /home/ HTTP/1.1" 200 1735
django.server WARNING 2025-03-13 10:33:18,766 basehttp 16944 22464 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.server INFO 2025-03-13 10:33:19,746 basehttp 16944 17264 "GET /tasks/ HTTP/1.1" 200 3713
django.server WARNING 2025-03-13 10:33:19,796 basehttp 16944 17264 "GET /static/css/styles.css HTTP/1.1" 404 1969
django.utils.autoreload INFO 2025-03-20 17:00:42,711 autoreload 23168 16436 Watching for file changes with StatReloader
django.server INFO 2025-03-20 17:00:44,135 basehttp 23168 2360 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 17:00:44,206 basehttp 23168 2360 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-03-20 17:00:44,433 basehttp 23168 2360 "GET /static/css/styles.css HTTP/1.1" 200 180
django.request WARNING 2025-03-20 17:00:44,545 log 23168 2360 Not Found: /favicon.ico
django.server WARNING 2025-03-20 17:00:44,545 basehttp 23168 2360 "GET /favicon.ico HTTP/1.1" 404 4578
django INFO 2025-03-20 17:00:55,907 views 23168 2360 User logged in: superadmin
django.server INFO 2025-03-20 17:00:55,933 basehttp 23168 2360 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 17:00:55,942 basehttp 23168 2360 "GET /home/ HTTP/1.1" 200 1795
django.server INFO 2025-03-20 17:01:04,978 basehttp 23168 2360 "GET /admin/ HTTP/1.1" 200 5296
django.server INFO 2025-03-20 17:01:05,024 basehttp 23168 2360 "GET /static/admin/css/base.css HTTP/1.1" 200 22092
django.server INFO 2025-03-20 17:01:05,026 basehttp 23168 14664 "GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
django.server INFO 2025-03-20 17:01:05,026 basehttp 23168 20560 "GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2804
django.server INFO 2025-03-20 17:01:05,026 basehttp 23168 21508 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
django.server INFO 2025-03-20 17:01:05,026 basehttp 23168 24156 "GET /static/admin/js/theme.js HTTP/1.1" 200 1653
django.server INFO 2025-03-20 17:01:05,027 basehttp 23168 18060 "GET /static/admin/css/responsive.css HTTP/1.1" 200 17972
django.server INFO 2025-03-20 17:01:05,126 basehttp 23168 18060 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
django.server INFO 2025-03-20 17:01:05,126 basehttp 23168 21508 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
django.server INFO 2025-03-20 17:01:05,127 basehttp 23168 24156 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
django.server INFO 2025-03-20 17:01:07,307 basehttp 23168 21508 "GET /admin/auth/group/ HTTP/1.1" 200 7093
django.server INFO 2025-03-20 17:01:07,338 basehttp 23168 21508 "GET /static/admin/css/changelists.css HTTP/1.1" 200 6878
django.server INFO 2025-03-20 17:01:07,360 basehttp 23168 24156 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:01:07,360 basehttp 23168 18060 "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
django.server INFO 2025-03-20 17:01:07,361 basehttp 23168 14664 "GET /static/admin/js/core.js HTTP/1.1" 200 6208
django.server INFO 2025-03-20 17:01:07,361 basehttp 23168 2360 "GET /static/admin/js/actions.js HTTP/1.1" 200 8076
django.server INFO 2025-03-20 17:01:07,361 basehttp 23168 20560 "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 9097
django.server INFO 2025-03-20 17:01:07,365 basehttp 23168 24156 "GET /static/admin/js/urlify.js HTTP/1.1" 200 7887
django.server INFO 2025-03-20 17:01:07,365 basehttp 23168 20560 "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531
django.server INFO 2025-03-20 17:01:07,367 basehttp 23168 21508 "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 285314
django.server INFO 2025-03-20 17:01:07,373 basehttp 23168 21508 "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 325171
django.server INFO 2025-03-20 17:01:07,422 basehttp 23168 21508 "GET /static/admin/img/search.svg HTTP/1.1" 200 458
django.server INFO 2025-03-20 17:01:07,469 basehttp 23168 21508 "GET /static/admin/js/filters.js HTTP/1.1" 200 978
django.server INFO 2025-03-20 17:01:07,486 basehttp 23168 21508 "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
django.server INFO 2025-03-20 17:01:10,241 basehttp 23168 21508 "GET /admin/auth/ HTTP/1.1" 200 5316
django.server INFO 2025-03-20 17:01:11,761 basehttp 23168 21508 "GET /admin/auth/group/ HTTP/1.1" 200 7093
django.server INFO 2025-03-20 17:01:11,807 basehttp 23168 21508 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:01:13,705 basehttp 23168 21508 "GET /admin/auth/ HTTP/1.1" 200 5316
django.server INFO 2025-03-20 17:01:14,366 basehttp 23168 21508 "GET /admin/ HTTP/1.1" 200 5296
django.server INFO 2025-03-20 17:01:16,163 basehttp 23168 21508 "GET /admin/auth/group/add/ HTTP/1.1" 200 12031
django.server INFO 2025-03-20 17:01:16,209 basehttp 23168 2360 "GET /static/admin/js/SelectBox.js HTTP/1.1" 200 4530
django.server INFO 2025-03-20 17:01:16,209 basehttp 23168 20560 "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 586
django.server INFO 2025-03-20 17:01:16,210 basehttp 23168 14664 "GET /static/admin/js/SelectFilter2.js HTTP/1.1" 200 15502
django.server INFO 2025-03-20 17:01:16,210 basehttp 23168 21508 "GET /static/admin/css/forms.css HTTP/1.1" 200 8794
django.server INFO 2025-03-20 17:01:16,213 basehttp 23168 24156 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:01:16,316 basehttp 23168 21508 "GET /static/admin/js/change_form.js HTTP/1.1" 200 606
django.server INFO 2025-03-20 17:01:16,317 basehttp 23168 24156 "GET /static/admin/css/widgets.css HTTP/1.1" 200 11564
django.server INFO 2025-03-20 17:01:16,350 basehttp 23168 24156 "GET /static/admin/img/icon-unknown.svg HTTP/1.1" 200 655
django.server INFO 2025-03-20 17:01:16,351 basehttp 23168 14664 "GET /static/admin/img/icon-unknown-alt.svg HTTP/1.1" 200 655
django.server INFO 2025-03-20 17:01:16,351 basehttp 23168 21508 "GET /static/admin/img/selector-icons.svg HTTP/1.1" 200 3291
django.server INFO 2025-03-20 17:01:55,058 basehttp 23168 21508 "GET /admin/auth/ HTTP/1.1" 200 5316
django.server INFO 2025-03-20 17:01:56,355 basehttp 23168 21508 "GET /admin/ HTTP/1.1" 200 5296
django.utils.autoreload INFO 2025-03-20 17:06:55,033 autoreload 23168 16436 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\models.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:06:55,790 autoreload 14176 5560 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 17:07:01,980 autoreload 12820 5368 Watching for file changes with StatReloader
django.server INFO 2025-03-20 17:07:03,138 basehttp 12820 2856 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 17:07:03,166 basehttp 12820 2856 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-03-20 17:07:03,291 basehttp 12820 2856 "GET /static/css/styles.css HTTP/1.1" 304 0
django INFO 2025-03-20 17:07:21,464 views 12820 2856 User logged in: superadmin
django.server INFO 2025-03-20 17:07:21,482 basehttp 12820 2856 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 17:07:21,493 basehttp 12820 2856 "GET /home/ HTTP/1.1" 200 1795
django.server INFO 2025-03-20 17:07:25,396 basehttp 12820 2856 "GET /admin/ HTTP/1.1" 200 5296
django.server INFO 2025-03-20 17:07:32,196 basehttp 12820 2856 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 17:07:32,211 basehttp 12820 2856 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-03-20 17:07:33,825 basehttp 12820 2856 "GET /admin/ HTTP/1.1" 200 5296
django.server INFO 2025-03-20 17:07:39,048 basehttp 12820 2856 "GET /admin/auth/group/ HTTP/1.1" 200 7093
django.server INFO 2025-03-20 17:07:39,085 basehttp 12820 2856 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.utils.autoreload INFO 2025-03-20 17:09:14,676 autoreload 12820 5368 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\home\admin.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:09:15,513 autoreload 25948 4092 Watching for file changes with StatReloader
django.server INFO 2025-03-20 17:09:19,189 basehttp 25948 12644 "GET /admin/auth/group/ HTTP/1.1" 200 7741
django.server INFO 2025-03-20 17:09:19,218 basehttp 25948 12644 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:09:21,893 basehttp 25948 12644 "GET /admin/home/customuser/add/ HTTP/1.1" 200 25536
django.server INFO 2025-03-20 17:09:21,928 basehttp 25948 17420 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:09:22,055 basehttp 25948 12644 "GET /static/admin/js/calendar.js HTTP/1.1" 200 9141
django.server INFO 2025-03-20 17:09:22,056 basehttp 25948 23776 "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 19319
django.server INFO 2025-03-20 17:09:22,156 basehttp 25948 23776 "GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086
django.server INFO 2025-03-20 17:09:22,156 basehttp 25948 12644 "GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677
django.server INFO 2025-03-20 17:09:47,840 basehttp 25948 12644 "GET /admin/auth/group/add/ HTTP/1.1" 200 12679
django.server INFO 2025-03-20 17:09:47,871 basehttp 25948 12644 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:10:15,411 basehttp 25948 12644 "GET /admin/home/customuser/add/ HTTP/1.1" 200 25536
django.server INFO 2025-03-20 17:10:15,439 basehttp 25948 12644 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:10:18,601 basehttp 25948 12644 "GET /admin/auth/group/add/?_to_field=id&_popup=1 HTTP/1.1" 200 8854
django.server INFO 2025-03-20 17:10:18,625 basehttp 25948 12644 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:10:32,934 basehttp 25948 12644 "GET /admin/home/customuser/ HTTP/1.1" 200 10781
django.server INFO 2025-03-20 17:10:32,977 basehttp 25948 12644 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:10:35,163 basehttp 25948 12644 "GET /admin/home/customuser/10/change/ HTTP/1.1" 200 25954
django.server INFO 2025-03-20 17:10:35,192 basehttp 25948 12644 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:10:46,647 basehttp 25948 12644 "POST /admin/home/customuser/10/change/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 17:10:46,698 basehttp 25948 12644 "GET /admin/home/customuser/ HTTP/1.1" 200 10986
django.server INFO 2025-03-20 17:10:46,805 basehttp 25948 12644 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:10:46,932 basehttp 25948 12644 "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
django.server INFO 2025-03-20 17:10:50,611 basehttp 25948 12644 "GET /admin/auth/group/ HTTP/1.1" 200 7741
django.server INFO 2025-03-20 17:10:50,639 basehttp 25948 12644 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:10:51,991 basehttp 25948 12644 "GET /admin/home/customuser/ HTTP/1.1" 200 10781
django.server INFO 2025-03-20 17:10:52,017 basehttp 25948 12644 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.utils.autoreload INFO 2025-03-20 17:12:15,043 autoreload 25948 4092 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\collectables\admin.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:12:15,879 autoreload 15916 18164 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 17:12:36,050 autoreload 15916 18164 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\collectables\admin.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:12:36,815 autoreload 21376 23592 Watching for file changes with StatReloader
django.server INFO 2025-03-20 17:12:38,245 basehttp 21376 11780 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 17:12:38,286 basehttp 21376 11780 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-03-20 17:12:38,461 basehttp 21376 11780 "GET /static/css/styles.css HTTP/1.1" 304 0
django.server INFO 2025-03-20 17:12:41,229 basehttp 21376 11780 "GET /admin/home/customuser/ HTTP/1.1" 200 11505
django.server INFO 2025-03-20 17:12:41,281 basehttp 21376 11780 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:12:43,428 basehttp 21376 11780 "GET /admin/collectables/collectable/ HTTP/1.1" 200 8186
django.server INFO 2025-03-20 17:12:43,455 basehttp 21376 11780 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:12:45,797 basehttp 21376 11780 "GET /admin/collectables/collectable/add/ HTTP/1.1" 200 10203
django.server INFO 2025-03-20 17:12:45,826 basehttp 21376 11780 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:12:49,858 basehttp 21376 11780 "GET /admin/collectables/collectable/ HTTP/1.1" 200 8186
django.utils.autoreload INFO 2025-03-20 17:13:57,200 autoreload 21376 23592 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\tasks\admin.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:13:57,992 autoreload 5432 10108 Watching for file changes with StatReloader
django.server INFO 2025-03-20 17:14:01,383 basehttp 5432 25848 "GET /admin/collectables/collectable/ HTTP/1.1" 200 8818
django.server INFO 2025-03-20 17:14:01,410 basehttp 5432 25848 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:14:04,024 basehttp 5432 25848 "GET /admin/tasks/tasks/ HTTP/1.1" 200 13017
django.server INFO 2025-03-20 17:14:04,056 basehttp 5432 25848 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:14:13,685 basehttp 5432 25848 "GET /admin/tasks/tasks/11/change/ HTTP/1.1" 200 14995
django.server INFO 2025-03-20 17:14:13,727 basehttp 5432 25848 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:14:22,388 basehttp 5432 25848 "GET /admin/collectables/collectable/ HTTP/1.1" 200 8818
django.server INFO 2025-03-20 17:14:22,417 basehttp 5432 25848 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:14:23,617 basehttp 5432 25848 "GET /admin/home/customuser/ HTTP/1.1" 200 12137
django.server INFO 2025-03-20 17:14:23,659 basehttp 5432 25848 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.utils.autoreload INFO 2025-03-20 17:15:25,828 autoreload 5432 10108 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\admin.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:15:26,659 autoreload 18676 1060 Watching for file changes with StatReloader
django.server INFO 2025-03-20 17:15:30,352 basehttp 18676 26500 "GET /admin/home/customuser/ HTTP/1.1" 200 12137
django.server INFO 2025-03-20 17:15:30,384 basehttp 18676 26500 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.utils.autoreload INFO 2025-03-20 17:15:30,388 autoreload 18676 1060 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\models.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:15:31,265 autoreload 11972 22220 Watching for file changes with StatReloader
django.server INFO 2025-03-20 17:15:31,688 basehttp 11972 26508 "GET /admin/home/customuser/ HTTP/1.1" 200 12137
django.server INFO 2025-03-20 17:15:31,720 basehttp 11972 26508 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.utils.autoreload INFO 2025-03-20 17:17:24,530 autoreload 23760 11808 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 17:21:51,168 autoreload 23760 11808 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\admin.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:21:51,924 autoreload 21688 3176 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 17:22:02,869 autoreload 18196 15912 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 17:22:47,362 autoreload 18196 15912 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\models.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:22:48,119 autoreload 26240 1908 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 17:23:00,840 autoreload 26240 1908 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:23:01,618 autoreload 772 11968 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 17:24:08,305 autoreload 772 11968 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\urls.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:24:09,038 autoreload 1988 10772 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 17:25:51,398 autoreload 1988 10772 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\models.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 17:25:52,156 autoreload 26176 11756 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 17:26:18,564 autoreload 6332 4356 Watching for file changes with StatReloader
django.server INFO 2025-03-20 17:26:20,557 basehttp 6332 23748 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 17:26:20,594 basehttp 6332 23748 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-03-20 17:26:20,709 basehttp 6332 23748 "GET /static/css/styles.css HTTP/1.1" 304 0
django INFO 2025-03-20 17:26:31,938 views 6332 23748 User logged in: superadmin
django.server INFO 2025-03-20 17:26:31,958 basehttp 6332 23748 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 17:26:31,969 basehttp 6332 23748 "GET /home/ HTTP/1.1" 200 1795
django.server INFO 2025-03-20 17:26:35,904 basehttp 6332 23748 "GET /admin/ HTTP/1.1" 200 10368
django.server INFO 2025-03-20 17:26:40,377 basehttp 6332 23748 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 10412
django.server INFO 2025-03-20 17:26:40,428 basehttp 6332 23748 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:26:42,125 basehttp 6332 23748 "GET /admin/unlockables/item/ HTTP/1.1" 200 10358
django.server INFO 2025-03-20 17:26:42,149 basehttp 6332 23748 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:26:43,700 basehttp 6332 23748 "GET /admin/unlockables/building/ HTTP/1.1" 200 10386
django.server INFO 2025-03-20 17:26:43,729 basehttp 6332 23748 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:26:45,422 basehttp 6332 23748 "GET /admin/unlockables/item/ HTTP/1.1" 200 10358
django.server INFO 2025-03-20 17:26:45,449 basehttp 6332 23748 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:26:46,807 basehttp 6332 23748 "GET /admin/unlockables/item/add/ HTTP/1.1" 200 13012
django.server INFO 2025-03-20 17:26:46,854 basehttp 6332 23748 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:26:51,884 basehttp 6332 23748 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 10412
django.server INFO 2025-03-20 17:26:51,911 basehttp 6332 23748 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:26:53,618 basehttp 6332 23748 "GET /admin/home/customuser/ HTTP/1.1" 200 13730
django.server INFO 2025-03-20 17:26:53,645 basehttp 6332 23748 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:26:59,393 basehttp 6332 23748 "GET /admin/home/customuser/6/change/ HTTP/1.1" 200 31324
django.server INFO 2025-03-20 17:26:59,422 basehttp 6332 23748 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:26:59,533 basehttp 6332 23748 "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
django.server INFO 2025-03-20 17:26:59,539 basehttp 6332 7156 "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
django.server INFO 2025-03-20 17:27:20,381 basehttp 6332 7156 "GET /admin/collectables/collectable/add/ HTTP/1.1" 200 12428
django.server INFO 2025-03-20 17:27:20,410 basehttp 6332 7156 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:27:24,546 basehttp 6332 7156 "GET /admin/unlockables/ HTTP/1.1" 200 6561
django.server INFO 2025-03-20 17:27:25,934 basehttp 6332 7156 "GET /admin/collectables/collectable/add/ HTTP/1.1" 200 12428
django.server INFO 2025-03-20 17:27:27,535 basehttp 6332 7156 "GET /admin/unlockables/item/ HTTP/1.1" 200 10358
django.server INFO 2025-03-20 17:27:27,564 basehttp 6332 7156 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:27:32,286 basehttp 6332 7156 "GET /admin/unlockables/item/add/ HTTP/1.1" 200 13012
django.server INFO 2025-03-20 17:27:32,313 basehttp 6332 7156 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:27:34,472 basehttp 6332 7156 "GET /admin/unlockables/item/ HTTP/1.1" 200 10358
django.server INFO 2025-03-20 17:27:35,926 basehttp 6332 7156 "GET /admin/unlockables/item/add/ HTTP/1.1" 200 13012
django.server INFO 2025-03-20 17:27:35,953 basehttp 6332 7156 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 17:27:37,615 basehttp 6332 7156 "GET /admin/unlockables/item/ HTTP/1.1" 200 10358
django.server INFO 2025-03-20 18:05:47,261 basehttp 6332 5368 "GET /admin/unlockables/building/ HTTP/1.1" 200 10386
django.server INFO 2025-03-20 18:05:47,295 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:08:42,086 basehttp 6332 5368 "GET /admin/home/customuser/ HTTP/1.1" 200 13730
django.server INFO 2025-03-20 18:08:42,117 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:08:46,874 basehttp 6332 5368 "GET /admin/home/customuser/9/change/ HTTP/1.1" 200 31316
django.server INFO 2025-03-20 18:08:46,908 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:08:56,674 basehttp 6332 5368 "GET /admin/collectables/collectable/add/?_to_field=id&_popup=1 HTTP/1.1" 200 5640
django.server INFO 2025-03-20 18:08:56,694 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:09:46,628 basehttp 6332 5368 "GET /admin/home/customuser/ HTTP/1.1" 200 13730
django.server INFO 2025-03-20 18:09:47,316 basehttp 6332 5368 "GET /admin/unlockables/building/ HTTP/1.1" 200 10386
django.server INFO 2025-03-20 18:09:47,861 basehttp 6332 5368 "GET /admin/unlockables/item/ HTTP/1.1" 200 10358
django.server INFO 2025-03-20 18:09:48,642 basehttp 6332 5368 "GET /admin/collectables/collectable/add/ HTTP/1.1" 200 12428
django.server INFO 2025-03-20 18:09:49,221 basehttp 6332 5368 "GET /admin/home/customuser/6/change/ HTTP/1.1" 200 31324
django.server INFO 2025-03-20 18:09:56,238 basehttp 6332 5368 "GET /admin/ HTTP/1.1" 200 10368
django.server INFO 2025-03-20 18:09:57,719 basehttp 6332 5368 "GET /home/ HTTP/1.1" 200 1795
django.server INFO 2025-03-20 18:09:57,762 basehttp 6332 5368 "GET /static/css/styles.css HTTP/1.1" 304 0
django.server INFO 2025-03-20 18:09:59,247 basehttp 6332 5368 "GET /collectables/ HTTP/1.1" 200 255
django.server INFO 2025-03-20 18:10:21,668 basehttp 6332 5368 "GET /admin/ HTTP/1.1" 200 10368
django.server INFO 2025-03-20 18:10:23,293 basehttp 6332 5368 "GET /admin/collectables/collectable/add/ HTTP/1.1" 200 12428
django.server INFO 2025-03-20 18:10:23,343 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:10:26,807 basehttp 6332 5368 "GET /admin/collectables/collectable/ HTTP/1.1" 200 10411
django.server INFO 2025-03-20 18:10:26,851 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:10:29,352 basehttp 6332 5368 "GET /admin/home/customuser/ HTTP/1.1" 200 13730
django.server INFO 2025-03-20 18:10:29,380 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:10:30,685 basehttp 6332 5368 "GET /admin/home/customuser/10/change/ HTTP/1.1" 200 31352
django.server INFO 2025-03-20 18:10:30,713 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:10:36,238 basehttp 6332 5368 "GET /admin/collectables/collectable/add/?_to_field=id&_popup=1 HTTP/1.1" 200 5640
django.server INFO 2025-03-20 18:10:36,259 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.request WARNING 2025-03-20 18:11:08,595 log 6332 5368 Not Found: /buy/
django.server WARNING 2025-03-20 18:11:08,596 basehttp 6332 5368 "GET /buy/ HTTP/1.1" 404 4836
django.request WARNING 2025-03-20 18:11:20,028 log 6332 5368 Not Found: /buy/item/
django.server WARNING 2025-03-20 18:11:20,029 basehttp 6332 5368 "GET /buy/item/ HTTP/1.1" 404 4851
django.server INFO 2025-03-20 18:15:32,931 basehttp 6332 5368 "GET /admin/home/customuser/10/change/ HTTP/1.1" 200 31352
django.server INFO 2025-03-20 18:18:52,662 basehttp 6332 5368 "GET /admin/collectables/collectable/ HTTP/1.1" 200 10411
django.server INFO 2025-03-20 18:18:52,690 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:19:07,133 basehttp 6332 5368 "GET /admin/unlockables/item/add/ HTTP/1.1" 200 13012
django.server INFO 2025-03-20 18:19:07,161 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:21:29,590 basehttp 6332 5368 "POST /admin/unlockables/item/add/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 18:21:29,645 basehttp 6332 5368 "GET /admin/unlockables/item/ HTTP/1.1" 200 11858
django.server INFO 2025-03-20 18:21:29,771 basehttp 6332 5368 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:21:42,160 basehttp 6332 5368 "GET /admin/ HTTP/1.1" 200 10722
django.request WARNING 2025-03-20 18:21:49,207 log 6332 5368 Not Found: /buy
django.server WARNING 2025-03-20 18:21:49,208 basehttp 6332 5368 "GET /buy HTTP/1.1" 404 4833
django.server INFO 2025-03-20 18:21:57,147 basehttp 6332 5368 "GET /buy/item/1 HTTP/1.1" 301 0
django.request ERROR 2025-03-20 18:21:57,211 log 6332 26604 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 9, in buy_item
    profile = request.user.userprofile
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\fields\related_descriptors.py", line 531, in __get__
    raise self.RelatedObjectDoesNotExist(
    ...<2 lines>...
    )
home.models.CustomUser.userprofile.RelatedObjectDoesNotExist: CustomUser has no userprofile.
django.server ERROR 2025-03-20 18:21:57,218 basehttp 6332 26604 "GET /buy/item/1/ HTTP/1.1" 500 83241
django.request ERROR 2025-03-20 18:22:01,782 log 6332 26604 Internal Server Error: /buy/item/0/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 8, in buy_item
    item = Item.objects.get(id=item_id)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 649, in get
    raise self.model.DoesNotExist(
        "%s matching query does not exist." % self.model._meta.object_name
    )
unlockables.models.Item.DoesNotExist: Item matching query does not exist.
django.server ERROR 2025-03-20 18:22:01,786 basehttp 6332 26604 "GET /buy/item/0/ HTTP/1.1" 500 83009
django.server INFO 2025-03-20 18:22:07,943 basehttp 6332 26604 "GET /admin/ HTTP/1.1" 200 10722
django.server INFO 2025-03-20 18:22:12,274 basehttp 6332 26604 "GET /admin/unlockables/item/ HTTP/1.1" 200 11650
django.server INFO 2025-03-20 18:22:12,321 basehttp 6332 26604 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:22:13,987 basehttp 6332 26604 "GET /admin/unlockables/item/1/change/ HTTP/1.1" 200 13444
django.server INFO 2025-03-20 18:22:14,029 basehttp 6332 26604 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:22:23,839 basehttp 6332 26604 "GET /admin/unlockables/item/ HTTP/1.1" 200 11650
django.server INFO 2025-03-20 18:22:25,960 basehttp 6332 26604 "GET /admin/ HTTP/1.1" 200 10722
django.server INFO 2025-03-20 18:22:52,023 basehttp 6332 26604 "GET /admin/unlockables/item/1/change/ HTTP/1.1" 200 13444
django.server INFO 2025-03-20 18:22:52,057 basehttp 6332 26604 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:22:56,484 basehttp 6332 26604 "POST /admin/unlockables/item/1/change/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 18:22:56,531 basehttp 6332 26604 "GET /admin/unlockables/item/ HTTP/1.1" 200 11860
django.server INFO 2025-03-20 18:22:56,771 basehttp 6332 26604 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 18:22:58,752 basehttp 6332 26604 "GET /admin/unlockables/item/1/change/ HTTP/1.1" 200 13444
django.server INFO 2025-03-20 18:22:59,306 basehttp 6332 26604 "GET /admin/ HTTP/1.1" 200 11081
django.server INFO 2025-03-20 18:22:59,920 basehttp 6332 26604 "GET /admin/unlockables/item/ HTTP/1.1" 200 11650
django.server INFO 2025-03-20 18:23:02,258 basehttp 6332 26604 "GET /admin/unlockables/item/add/ HTTP/1.1" 200 13012
django.server INFO 2025-03-20 18:23:02,953 basehttp 6332 26604 "GET /admin/collectables/collectable/ HTTP/1.1" 200 10411
django.server INFO 2025-03-20 18:23:03,535 basehttp 6332 26604 "GET /admin/home/customuser/10/change/ HTTP/1.1" 200 31352
django.server INFO 2025-03-20 18:23:03,870 basehttp 6332 26604 "GET /admin/home/customuser/ HTTP/1.1" 200 13730
django.server INFO 2025-03-20 18:23:04,410 basehttp 6332 26604 "GET /admin/collectables/collectable/ HTTP/1.1" 200 10411
django.server INFO 2025-03-20 18:23:04,892 basehttp 6332 26604 "GET /admin/ HTTP/1.1" 200 11081
django.request ERROR 2025-03-20 18:23:21,998 log 6332 26604 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 9, in buy_item
    profile = request.user.userprofile
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\fields\related_descriptors.py", line 531, in __get__
    raise self.RelatedObjectDoesNotExist(
    ...<2 lines>...
    )
home.models.CustomUser.userprofile.RelatedObjectDoesNotExist: CustomUser has no userprofile.
django.server ERROR 2025-03-20 18:23:22,003 basehttp 6332 26604 "GET /buy/item/1/ HTTP/1.1" 500 83241
django.request ERROR 2025-03-20 18:23:36,755 log 6332 26604 Internal Server Error: /buy/item/0/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 8, in buy_item
    item = Item.objects.get(id=item_id)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 649, in get
    raise self.model.DoesNotExist(
        "%s matching query does not exist." % self.model._meta.object_name
    )
unlockables.models.Item.DoesNotExist: Item matching query does not exist.
django.server ERROR 2025-03-20 18:23:36,758 basehttp 6332 26604 "GET /buy/item/0/ HTTP/1.1" 500 83009
django.request ERROR 2025-03-20 18:23:40,096 log 6332 26604 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 9, in buy_item
    profile = request.user.userprofile
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\fields\related_descriptors.py", line 531, in __get__
    raise self.RelatedObjectDoesNotExist(
    ...<2 lines>...
    )
home.models.CustomUser.userprofile.RelatedObjectDoesNotExist: CustomUser has no userprofile.
django.server ERROR 2025-03-20 18:23:40,100 basehttp 6332 26604 "GET /buy/item/1/ HTTP/1.1" 500 83241
django.utils.autoreload INFO 2025-03-20 19:04:51,043 autoreload 19556 23728 Watching for file changes with StatReloader
django.server INFO 2025-03-20 19:04:52,269 basehttp 19556 11784 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:04:52,352 basehttp 19556 11784 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-03-20 19:04:52,634 basehttp 19556 11784 "GET /static/css/styles.css HTTP/1.1" 304 0
django INFO 2025-03-20 19:05:04,159 views 19556 11784 User logged in: superadmin
django.server INFO 2025-03-20 19:05:04,189 basehttp 19556 11784 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:05:04,198 basehttp 19556 11784 "GET /home/ HTTP/1.1" 200 1795
django.server INFO 2025-03-20 19:05:07,141 basehttp 19556 11784 "GET /admin/ HTTP/1.1" 200 11081
django.server INFO 2025-03-20 19:05:11,974 basehttp 19556 11784 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 10412
django.server INFO 2025-03-20 19:05:12,026 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:05:14,020 basehttp 19556 11784 "GET /admin/unlockables/userprofile/add/ HTTP/1.1" 200 14566
django.server INFO 2025-03-20 19:05:14,064 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:05:50,786 basehttp 19556 11784 "GET /admin/unlockables/building/add/?_to_field=id&_popup=1 HTTP/1.1" 200 5675
django.server INFO 2025-03-20 19:05:50,810 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:06:09,841 basehttp 19556 11784 "POST /admin/unlockables/userprofile/add/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:06:09,894 basehttp 19556 11784 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 11971
django.server INFO 2025-03-20 19:06:10,016 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.request ERROR 2025-03-20 19:06:25,495 log 19556 11784 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 9, in buy_item
    profile = request.user.userprofile
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\fields\related_descriptors.py", line 531, in __get__
    raise self.RelatedObjectDoesNotExist(
    ...<2 lines>...
    )
home.models.CustomUser.userprofile.RelatedObjectDoesNotExist: CustomUser has no userprofile.
django.server ERROR 2025-03-20 19:06:25,501 basehttp 19556 11784 "GET /buy/item/1/ HTTP/1.1" 500 83241
django.server INFO 2025-03-20 19:06:42,021 basehttp 19556 11784 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 11741
django.server INFO 2025-03-20 19:06:45,532 basehttp 19556 11784 "GET /admin/unlockables/userprofile/1/change/ HTTP/1.1" 200 14833
django.server INFO 2025-03-20 19:06:45,563 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:08:54,978 basehttp 19556 11784 "POST /admin/unlockables/userprofile/1/change/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:08:55,028 basehttp 19556 11784 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 11973
django.server INFO 2025-03-20 19:08:55,256 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:10:06,612 basehttp 19556 11784 "GET /admin/home/customuser/ HTTP/1.1" 200 13730
django.server INFO 2025-03-20 19:10:06,641 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:10:11,358 basehttp 19556 11784 "GET /admin/home/customuser/9/change/ HTTP/1.1" 200 31316
django.server INFO 2025-03-20 19:10:11,388 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:10:24,326 basehttp 19556 11784 "GET /admin/home/customuser/ HTTP/1.1" 200 13730
django.server INFO 2025-03-20 19:10:27,776 basehttp 19556 11784 "GET /admin/home/customuser/10/change/ HTTP/1.1" 200 31352
django.server INFO 2025-03-20 19:10:27,820 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:10:38,222 basehttp 19556 11784 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 11741
django.server INFO 2025-03-20 19:10:38,247 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:10:40,500 basehttp 19556 11784 "GET /admin/unlockables/userprofile/1/change/ HTTP/1.1" 200 14833
django.server INFO 2025-03-20 19:10:40,541 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:10:46,187 basehttp 19556 11784 "GET /admin/unlockables/userprofile/add/ HTTP/1.1" 200 14566
django.server INFO 2025-03-20 19:10:46,216 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:10:48,089 basehttp 19556 11784 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 11741
django.server INFO 2025-03-20 19:10:48,116 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:10:57,317 basehttp 19556 11784 "GET /admin/unlockables/ HTTP/1.1" 200 6561
django.server INFO 2025-03-20 19:10:59,387 basehttp 19556 11784 "GET /admin/ HTTP/1.1" 200 11838
django.server INFO 2025-03-20 19:11:01,276 basehttp 19556 11784 "GET /admin/collectables/collectable/ HTTP/1.1" 200 10411
django.server INFO 2025-03-20 19:11:01,310 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:03,224 basehttp 19556 11784 "GET /admin/home/customuser/ HTTP/1.1" 200 13730
django.server INFO 2025-03-20 19:11:03,251 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:04,050 basehttp 19556 11784 "GET /admin/tasks/tasks/ HTTP/1.1" 200 14610
django.server INFO 2025-03-20 19:11:04,077 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:06,238 basehttp 19556 11784 "GET /admin/tasks/tasks/11/change/ HTTP/1.1" 200 16588
django.server INFO 2025-03-20 19:11:06,265 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:15,616 basehttp 19556 11784 "GET /admin/unlockables/building/ HTTP/1.1" 200 10386
django.server INFO 2025-03-20 19:11:15,646 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:18,056 basehttp 19556 11784 "GET /admin/unlockables/item/ HTTP/1.1" 200 11650
django.server INFO 2025-03-20 19:11:18,083 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:19,719 basehttp 19556 11784 "GET /admin/unlockables/item/1/change/ HTTP/1.1" 200 13444
django.server INFO 2025-03-20 19:11:19,745 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:24,416 basehttp 19556 11784 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 11741
django.server INFO 2025-03-20 19:11:24,446 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:27,256 basehttp 19556 11784 "GET /admin/home/customuser/ HTTP/1.1" 200 13730
django.server INFO 2025-03-20 19:11:27,282 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:29,265 basehttp 19556 11784 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 11741
django.server INFO 2025-03-20 19:11:29,307 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:31,504 basehttp 19556 11784 "GET /admin/unlockables/userprofile/1/change/ HTTP/1.1" 200 14833
django.server INFO 2025-03-20 19:11:31,534 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:37,815 basehttp 19556 11784 "GET /admin/unlockables/item/ HTTP/1.1" 200 11650
django.server INFO 2025-03-20 19:11:37,863 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:39,109 basehttp 19556 11784 "GET /admin/unlockables/item/1/change/ HTTP/1.1" 200 13444
django.server INFO 2025-03-20 19:11:39,137 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:41,039 basehttp 19556 11784 "GET /admin/unlockables/building/ HTTP/1.1" 200 10386
django.server INFO 2025-03-20 19:11:41,067 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:42,243 basehttp 19556 11784 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 11741
django.server INFO 2025-03-20 19:11:42,273 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:48,872 basehttp 19556 11784 "GET /admin/unlockables/userprofile/1/change/ HTTP/1.1" 200 14833
django.server INFO 2025-03-20 19:11:48,901 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:11:49,942 basehttp 19556 11784 "GET /admin/unlockables/building/add/?_to_field=id&_popup=1 HTTP/1.1" 200 5675
django.server INFO 2025-03-20 19:11:49,965 basehttp 19556 11784 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.utils.autoreload INFO 2025-03-20 19:12:10,754 autoreload 22116 4208 Watching for file changes with StatReloader
django.server INFO 2025-03-20 19:12:11,911 basehttp 22116 11552 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:12:11,945 basehttp 22116 11552 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-03-20 19:12:20,398 basehttp 22116 11552 "GET /admin/home/customuser/ HTTP/1.1" 200 13730
django.server INFO 2025-03-20 19:12:20,428 basehttp 22116 11552 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:12:26,027 basehttp 22116 11552 "GET /home/signup/ HTTP/1.1" 200 2544
django INFO 2025-03-20 19:12:41,596 views 22116 11552 User logged in: jacob
django INFO 2025-03-20 19:12:41,613 views 22116 11552 New user signed up: jacob
django.server INFO 2025-03-20 19:12:41,621 basehttp 22116 11552 "POST /home/signup/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:12:41,631 basehttp 22116 11552 "GET /home/ HTTP/1.1" 200 1790
django.request ERROR 2025-03-20 19:12:54,378 log 22116 11552 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 9, in buy_item
    profile = request.user.userprofile
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\fields\related_descriptors.py", line 531, in __get__
    raise self.RelatedObjectDoesNotExist(
    ...<2 lines>...
    )
home.models.CustomUser.userprofile.RelatedObjectDoesNotExist: CustomUser has no userprofile.
django.server ERROR 2025-03-20 19:12:54,385 basehttp 22116 11552 "GET /buy/item/1/ HTTP/1.1" 500 83495
django.request ERROR 2025-03-20 19:12:54,661 log 22116 11552 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 9, in buy_item
    profile = request.user.userprofile
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\fields\related_descriptors.py", line 531, in __get__
    raise self.RelatedObjectDoesNotExist(
    ...<2 lines>...
    )
home.models.CustomUser.userprofile.RelatedObjectDoesNotExist: CustomUser has no userprofile.
django.server ERROR 2025-03-20 19:12:54,666 basehttp 22116 11552 "GET /buy/item/1/ HTTP/1.1" 500 83221
django.server INFO 2025-03-20 19:12:59,598 basehttp 22116 11552 "GET /admin/unlockables/userprofile/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:12:59,629 basehttp 22116 11552 "GET /admin/login/?next=/admin/unlockables/userprofile/ HTTP/1.1" 200 3722
django.server INFO 2025-03-20 19:12:59,851 basehttp 22116 11552 "GET /static/admin/css/login.css HTTP/1.1" 200 951
django INFO 2025-03-20 19:13:14,749 views 22116 11552 User logged in: superadmin
django.server INFO 2025-03-20 19:13:14,771 basehttp 22116 11552 "POST /admin/login/?next=/admin/unlockables/userprofile/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:13:14,822 basehttp 22116 11552 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 11741
django.server INFO 2025-03-20 19:13:15,059 basehttp 22116 11552 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:13:18,512 basehttp 22116 11552 "GET /admin/unlockables/userprofile/ HTTP/1.1" 200 11741
django.server INFO 2025-03-20 19:13:18,541 basehttp 22116 11552 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:14:16,492 basehttp 22116 11552 "GET /admin/home/customuser/ HTTP/1.1" 200 13996
django.server INFO 2025-03-20 19:14:16,521 basehttp 22116 11552 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:14:34,780 basehttp 22116 11552 "GET /admin/home/customuser/11/change/ HTTP/1.1" 200 31309
django.server INFO 2025-03-20 19:14:34,808 basehttp 22116 11552 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.utils.autoreload INFO 2025-03-20 19:21:15,645 autoreload 19556 11108 Watching for file changes with StatReloader
django.server INFO 2025-03-20 19:21:17,150 basehttp 19556 21008 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:21:17,201 basehttp 19556 21008 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-03-20 19:21:17,270 basehttp 19556 21008 "GET /static/css/styles.css HTTP/1.1" 304 0
django.server INFO 2025-03-20 19:21:17,633 basehttp 19556 21008 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:21:17,643 basehttp 19556 21008 "GET /accounts/login/ HTTP/1.1" 200 1539
django INFO 2025-03-20 19:21:40,364 views 19556 21008 User logged in: superadmin
django.server INFO 2025-03-20 19:21:40,388 basehttp 19556 21008 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:21:40,400 basehttp 19556 21008 "GET /home/ HTTP/1.1" 200 1795
django.server INFO 2025-03-20 19:21:45,987 basehttp 19556 21008 "GET /admin/ HTTP/1.1" 200 10431
django.server INFO 2025-03-20 19:21:48,221 basehttp 19556 21008 "GET /admin/unlockables/item/ HTTP/1.1" 200 10731
django.server INFO 2025-03-20 19:21:48,276 basehttp 19556 21008 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:21:50,454 basehttp 19556 21008 "GET /admin/unlockables/item/1/change/ HTTP/1.1" 200 12525
django.server INFO 2025-03-20 19:21:50,483 basehttp 19556 21008 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.utils.autoreload INFO 2025-03-20 19:22:16,860 autoreload 19556 11108 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\admin.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 19:22:17,741 autoreload 11024 21816 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 19:39:40,229 autoreload 8656 10916 Watching for file changes with StatReloader
django.server INFO 2025-03-20 19:39:43,807 basehttp 8656 21812 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:39:43,842 basehttp 8656 21812 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-03-20 19:39:44,039 basehttp 8656 21812 "GET /static/css/styles.css HTTP/1.1" 304 0
django.server INFO 2025-03-20 19:39:44,254 basehttp 8656 21812 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:39:44,262 basehttp 8656 21812 "GET /accounts/login/ HTTP/1.1" 200 1539
django INFO 2025-03-20 19:39:57,156 views 8656 21812 User logged in: superadmin
django.server INFO 2025-03-20 19:39:57,180 basehttp 8656 21812 "POST /accounts/login/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:39:57,191 basehttp 8656 21812 "GET /home/ HTTP/1.1" 200 1795
django.server INFO 2025-03-20 19:40:03,887 basehttp 8656 21812 "GET /admin/ HTTP/1.1" 200 11107
django.server INFO 2025-03-20 19:40:05,923 basehttp 8656 21812 "GET /admin/unlockables/userinventory/ HTTP/1.1" 200 9988
django.server INFO 2025-03-20 19:40:05,973 basehttp 8656 21812 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:40:10,001 basehttp 8656 21812 "GET /admin/unlockables/userinventory/add/ HTTP/1.1" 200 14916
django.server INFO 2025-03-20 19:40:10,046 basehttp 8656 21812 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:40:19,239 basehttp 8656 21812 "POST /admin/unlockables/userinventory/add/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:40:19,290 basehttp 8656 21812 "GET /admin/unlockables/userinventory/ HTTP/1.1" 200 11557
django.server INFO 2025-03-20 19:40:19,516 basehttp 8656 21812 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:40:22,554 basehttp 8656 21812 "GET /admin/unlockables/userinventory/1/change/ HTTP/1.1" 200 15187
django.server INFO 2025-03-20 19:40:22,585 basehttp 8656 21812 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:40:23,948 basehttp 8656 21812 "GET /admin/unlockables/userinventory/1/delete/ HTTP/1.1" 200 10149
django.server INFO 2025-03-20 19:40:24,041 basehttp 8656 21812 "GET /static/admin/js/cancel.js HTTP/1.1" 200 884
django.server INFO 2025-03-20 19:40:32,581 basehttp 8656 21812 "POST /admin/unlockables/userinventory/1/delete/ HTTP/1.1" 302 0
django.server INFO 2025-03-20 19:40:32,623 basehttp 8656 21812 "GET /admin/unlockables/userinventory/ HTTP/1.1" 200 10167
django.server INFO 2025-03-20 19:40:32,774 basehttp 8656 21812 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.request ERROR 2025-03-20 19:40:45,043 log 8656 21812 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 9, in buy_item
    profile = request.user.userprofile
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
AttributeError: 'CustomUser' object has no attribute 'userprofile'
django.server ERROR 2025-03-20 19:40:45,049 basehttp 8656 21812 "GET /buy/item/1/ HTTP/1.1" 500 78824
django.request ERROR 2025-03-20 19:40:45,266 log 8656 21812 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 9, in buy_item
    profile = request.user.userprofile
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
AttributeError: 'CustomUser' object has no attribute 'userprofile'
django.server ERROR 2025-03-20 19:40:45,271 basehttp 8656 21812 "GET /buy/item/1/ HTTP/1.1" 500 78550
django.server INFO 2025-03-20 19:40:57,870 basehttp 8656 21812 "GET /admin/unlockables/userinventory/ HTTP/1.1" 200 9988
django.server INFO 2025-03-20 19:41:00,514 basehttp 8656 21812 "GET /admin/unlockables/userinventory/ HTTP/1.1" 200 9988
django.server INFO 2025-03-20 19:41:00,541 basehttp 8656 21812 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:41:01,534 basehttp 8656 21812 "GET /admin/home/customuser/ HTTP/1.1" 200 13558
django.server INFO 2025-03-20 19:41:01,572 basehttp 8656 21812 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 19:41:03,591 basehttp 8656 21812 "GET /admin/home/customuser/11/change/ HTTP/1.1" 200 31102
django.server INFO 2025-03-20 19:41:03,635 basehttp 8656 21812 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 20:57:43,479 basehttp 8656 2036 "GET /admin/unlockables/userinventory/ HTTP/1.1" 200 9988
django.server INFO 2025-03-20 20:57:43,521 basehttp 8656 2036 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 20:57:45,499 basehttp 8656 2036 "GET /admin/unlockables/userinventory/add/ HTTP/1.1" 200 14916
django.server INFO 2025-03-20 20:57:45,546 basehttp 8656 2036 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.request ERROR 2025-03-20 20:57:59,092 log 8656 2036 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 9, in buy_item
    profile = request.user.userprofile
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
AttributeError: 'CustomUser' object has no attribute 'userprofile'
django.server ERROR 2025-03-20 20:57:59,096 basehttp 8656 2036 "GET /buy/item/1/ HTTP/1.1" 500 78550
django.request ERROR 2025-03-20 20:58:27,338 log 8656 2036 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 9, in buy_item
    profile = request.user.User
              ^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
AttributeError: 'CustomUser' object has no attribute 'userprofile'
django.server ERROR 2025-03-20 20:58:27,342 basehttp 8656 2036 "GET /buy/item/1/ HTTP/1.1" 500 78673
django.utils.autoreload INFO 2025-03-20 20:58:27,515 autoreload 8656 10916 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 20:58:28,334 autoreload 18896 26104 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 20:58:36,561 autoreload 18896 26104 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 20:58:37,348 autoreload 27144 10244 Watching for file changes with StatReloader
django.request ERROR 2025-03-20 20:58:38,141 log 27144 12892 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 11, in buy_item
    if profile.carbo_coins >= item.price:
       ^^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'carbo_coins'
django.server ERROR 2025-03-20 20:58:38,146 basehttp 27144 12892 "GET /buy/item/1/ HTTP/1.1" 500 74757
django.request ERROR 2025-03-20 20:58:50,573 log 27144 12892 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 11, in buy_item
    if profile.carbo_coins >= item.price:
       ^^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'carbo_coins'
django.server ERROR 2025-03-20 20:58:50,577 basehttp 27144 12892 "GET /buy/item/1/ HTTP/1.1" 500 74748
django.utils.autoreload INFO 2025-03-20 20:58:50,628 autoreload 27144 10244 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 20:58:51,436 autoreload 27420 15428 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 20:58:54,756 autoreload 27420 15428 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 20:58:55,587 autoreload 3824 26104 Watching for file changes with StatReloader
django.request ERROR 2025-03-20 20:58:55,944 log 3824 7276 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 11, in buy_item
    if profile.carbo_coins >= item.price:
       ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
AttributeError: 'CustomUser' object has no attribute 'carbo_coins'
django.server ERROR 2025-03-20 20:58:55,950 basehttp 3824 7276 "GET /buy/item/1/ HTTP/1.1" 500 78848
django.request ERROR 2025-03-20 20:58:56,305 log 3824 7276 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 11, in buy_item
    if profile.carbo_coins >= item.price:
       ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
AttributeError: 'CustomUser' object has no attribute 'carbo_coins'
django.server ERROR 2025-03-20 20:58:56,310 basehttp 3824 7276 "GET /buy/item/1/ HTTP/1.1" 500 78848
django.server INFO 2025-03-20 20:59:04,506 basehttp 3824 13496 "GET /admin/unlockables/userinventory/add/ HTTP/1.1" 200 14916
django.server INFO 2025-03-20 20:59:06,186 basehttp 3824 13496 "GET /admin/home/customuser/ HTTP/1.1" 200 13558
django.server INFO 2025-03-20 20:59:06,214 basehttp 3824 13496 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.server INFO 2025-03-20 20:59:08,158 basehttp 3824 13496 "GET /admin/home/customuser/10/change/ HTTP/1.1" 200 31145
django.server INFO 2025-03-20 20:59:08,191 basehttp 3824 13496 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
django.utils.autoreload INFO 2025-03-20 20:59:32,119 autoreload 3824 26104 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 20:59:33,020 autoreload 10796 16208 Watching for file changes with StatReloader
django.server INFO 2025-03-20 20:59:34,608 basehttp 10796 10916 "GET /admin/home/customuser/ HTTP/1.1" 200 13558
django.request ERROR 2025-03-20 20:59:43,448 log 10796 10916 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 18, in buy_item
    return redirect('game_page')  # Redirect to the main game page
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\shortcuts.py", line 49, in redirect
    return redirect_class(resolve_url(to, *args, **kwargs))
                          ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\shortcuts.py", line 180, in resolve_url
    return reverse(to, args=args, kwargs=kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'game_page' not found. 'game_page' is not a valid view function or pattern name.
django.server ERROR 2025-03-20 20:59:43,453 basehttp 10796 10916 "GET /buy/item/1/ HTTP/1.1" 500 92675
django.request ERROR 2025-03-20 20:59:55,937 log 10796 10916 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 18, in buy_item
    return redirect('game')  # Redirect to the main game page
           ~~~~~~~~^^^^^^^^~~~~~
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\shortcuts.py", line 49, in redirect
    return redirect_class(resolve_url(to, *args, **kwargs))
                          ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\shortcuts.py", line 180, in resolve_url
    return reverse(to, args=args, kwargs=kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'game_page' not found. 'game_page' is not a valid view function or pattern name.
django.server ERROR 2025-03-20 20:59:55,940 basehttp 10796 10916 "GET /buy/item/1/ HTTP/1.1" 500 93053
django.utils.autoreload INFO 2025-03-20 20:59:56,629 autoreload 10796 16208 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 20:59:57,391 autoreload 25984 16588 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 21:00:00,685 autoreload 25984 16588 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 21:00:01,429 autoreload 21700 15396 Watching for file changes with StatReloader
django.request ERROR 2025-03-20 21:00:03,413 log 21700 18104 Internal Server Error: /buy/item/1/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 18, in buy_item
    return redirect('game')  # Redirect to the main game page
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\shortcuts.py", line 49, in redirect
    return redirect_class(resolve_url(to, *args, **kwargs))
                          ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\shortcuts.py", line 180, in resolve_url
    return reverse(to, args=args, kwargs=kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'game' not found. 'game' is not a valid view function or pattern name.
django.server ERROR 2025-03-20 21:00:03,418 basehttp 21700 18104 "GET /buy/item/1/ HTTP/1.1" 500 92979
django.utils.autoreload INFO 2025-03-20 21:00:14,649 autoreload 21700 15396 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 21:00:15,442 autoreload 956 26816 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 21:00:18,657 autoreload 956 26816 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\models.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 21:00:19,466 autoreload 22580 3176 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 21:03:38,279 autoreload 22580 3176 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 21:03:39,088 autoreload 21896 5612 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 21:04:09,682 autoreload 21896 5612 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\urls.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 21:04:10,456 autoreload 10736 26472 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 21:04:20,876 autoreload 10736 26472 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 21:04:21,627 autoreload 4932 10096 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 21:04:24,462 autoreload 4932 10096 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\urls.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 21:04:25,181 autoreload 26948 7576 Watching for file changes with StatReloader
django.utils.autoreload INFO 2025-03-20 21:04:37,814 autoreload 26948 7576 C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\urls.py changed, reloading.
django.utils.autoreload INFO 2025-03-20 21:04:38,549 autoreload 2932 9640 Watching for file changes with StatReloader
django.server INFO 2025-03-20 21:04:40,494 basehttp 2932 26472 "GET / HTTP/1.1" 302 0
django.server INFO 2025-03-20 21:04:40,531 basehttp 2932 26472 "GET /accounts/login/ HTTP/1.1" 200 1539
django.server INFO 2025-03-20 21:04:40,682 basehttp 2932 26472 "GET /static/css/styles.css HTTP/1.1" 304 0
django.request ERROR 2025-03-20 21:04:48,472 log 2932 26472 Internal Server Error: /shop/
Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ~~~~~~~~~~~~^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        template_name,
        ^^^^^^^^^^^^^^
        skip=history,
        ^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\engine.py", line 163, in find_template
    raise TemplateDoesNotExist(name, tried=tried)
django.template.exceptions.TemplateDoesNotExist: base.html

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Jacob\Group software\Group-Engineering-Project-6\SustainableApplication\unlockables\views.py", line 23, in shop
    return render(request, 'shop.html', {'items': items})
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\backends\django.py", line 109, in render
    reraise(exc, self.backend)
    ~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Jacob\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\template\backends\django.py", line 130, in reraise
    raise new from exc
django.template.exceptions.TemplateDoesNotExist: base.html
django.server ERROR 2025-03-20 21:04:48,479 basehttp 2932 26472 "GET /shop/ HTTP/1.1" 500 140403
